projects/wmf_default.yaml

```diff
@@ -83,11 +83,11 @@ paths:
             /feed:
               x-modules:
                 - path: v1/feed.js
-                  options: '{{merge({"feed_cache_control": "s-maxage=300, max-age=60", "ttl": 7200}, options.mobileapps)}}'
+                  options: '{{merge({"feed_cache_control": "s-maxage=300, max-age=60", "ttl": 300}, options.mobileapps)}}'
```

Comment:
The `ttl` is not used any more, so it can be safely removed (here and elsewhere).

Reply:
Done